n6001/hardware/ofs_n6001/build/scripts/run.sh

```diff
@@ -103,72 +108,20 @@ if [ -n "$OFS_ASP_ENV_ENABLE_ASE" ]; then
     exit $?
 fi
 
-echo "Calling compile_script"
-quartus_sh -t scripts/compile_script.tcl "$BSP_FLOW"
-FLOW_SUCCESS=$?
-
-# Report Timing
-# =============
-DO_ADJUST_PLLS=1
-PLL_METADATA_FILE="pll_metadata.txt"
-if [ $FLOW_SUCCESS -eq 0 ]
-then
-    if [ $DO_ADJUST_PLLS -eq 0 ]; then
-        echo "Not running adjust_plls.tcl. We still need to create a pll_metadata.txt file. Doing that now with 1x clock @ 150MHz and 2x clock @ 300 MHz."
-        echo "clock-frequency-low:150 clock-frequency-high:300" >> "$PLL_METADATA_FILE"
-    else
-        echo "Running adjust_plls.txt script to find the highest valid kernel_clock."
-        quartus_sh -t scripts/adjust_plls.tcl ofs_top "${BSP_FLOW}"
-    fi
-else
-    echo "ERROR: kernel compilation failed. Please see quartus_sh_compile.log for more information."
-    exit 1
-fi
-
-#run packager tool to create GBS
-BBS_ID_FILE="fme-ifc-id.txt"
-if [ -f "$BBS_ID_FILE" ]; then
-    FME_IFC_ID=$(cat $BBS_ID_FILE)
-    echo "FME_IFC_ID is $FME_IFC_ID"
-    cat $BBS_ID_FILE
-else
-    echo "ERROR: fme id not found."
-    exit 1
-fi
+echo "Starting: quartus_sh --flow compile $Q_REVISION -c $BSP_FLOW"
+quartus_sh --flow compile $Q_REVISION -c $BSP_FLOW
 
-if [ -f "$PLL_METADATA_FILE" ]; then
-    IFS=" " read -r -a PLL_METADATA <<< "$(cat $PLL_METADATA_FILE)"
-else
-    echo "Error: cannot find $PLL_METADATA_FILE"
-    exit 1
-fi
-
-#check for generated rbf and gbs files
-if [ ! -f "./output_files/${BSP_FLOW}.green_region.rbf" ]; then
-    echo "ERROR: ./output_files/${BSP_FLOW}.green_region.rbf is missing!"
-    exit 1
-fi
+rm -rf fpga.bin
 
-rm -f ./output_files/"${BSP_FLOW}".gbs
```

Comment:
we don't need to run packager on our own anymore. it happens magically during a post-processing tcl script as defined in the FIM's qsf/tcl files.

Reply:
we could also consider getting rid of the packager-check at the top of run.sh. That check is nice, however, because it would be annoying to find out the packager can't be found after waiting hours and hours for a build to complete.